content/sensu-go/5.0/installation/verify.md

```diff
@@ -24,21 +24,21 @@ You can verify a Sensu download using SHA-512 checksums.
 Download Sensu for Windows.
 
 {{< highlight text >}}
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.0.1/sensu-go-5.0.1-windows-amd64.tar.gz  -OutFile C:\Users\Administrator\sensu-go-5.0.1-windows-amd64.tar.gz
+Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.0.1/sensu-go-5.0.1-windows-amd64.tar.gz  -OutFile "$env:userprofile\sensu-go-5.0.1-windows-amd64.tar.gz"
 {{< /highlight >}}
 
 Generate a SHA-512 checksum for the downloaded artifact.
 
 {{< highlight text >}}
-Get-FileHash C:\Users\Administrator\sensu-go-5.0.1-windows-amd64.tar.gz -Algorithm SHA512 | Format-List
+Get-FileHash "$env:userprofile\sensu-go-5.0.1-windows-amd64.tar.gz" -Algorithm SHA512 | Format-List
 {{< /highlight >}}
 
 The result should match (with the exception of capitalization) the output from the following commands.
 
 {{< highlight text >}}
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.0.1/sensu-go-5.0.1-windows-amd64.sha512sum -OutFile C:\Users\Administrator\sensu-go-5.0.1-windows-amd64.sha512sum
+Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.0.1/sensu-go-5.0.1-windows-amd64.sha512sum -OutFile "$env:userprofile\sensu-go-5.0.1-windows-amd64.sha512sum"
 
-Get-Content C:\Users\Administrator\sensu-go-5.0.1-windows-amd64.sha512sum
+Get-Content "$env:userprofile\sensu-go-5.0.1-windows-amd64.sha512sum"
```

Comment:
I am having issues getting the computed check sum of the download and the listed checksum on S3 to match.

<img width="879" alt="screen shot 2019-02-14 at 10 49 41 pm" src="https://user-images.githubusercontent.com/19964713/52840255-8deb6300-30ad-11e9-82c2-045ca789636c.png">